defaults/main.yml

```diff
@@ -1,8 +1,24 @@
-# SPDX-License-Identifier: MIT
 ---
-# Here is the right place to put the role's input variables.
-# This file also serves as a documentation for such a variables.
+# Whether to change global configuration or per-user one
+ssh_user: null
+# Use default OS values or start from scratch
+# (for per-user configuration or included files in drop-in directories)
+ssh_skip_defaults: auto
+# The name to be used as part of filename in drop-in directory
+ssh_drop_in_name: false
+# dict of configuration options
+ssh: []
+# list of additional packages to install
+# for exmaple openssh-keycat or openssh-askpass
+ssh_additional_packages: []
 
-# Examples of role input variables:
-ssh_foo: "foo"
-ssh_bar: "bar"
+########## Internal defaults ##########
```

Comment:
These should go in `vars/main.yml` if they are not intended for users to use

Reply:
Moved and added missing. Also I changed some of the defaults from false to `none` as it makes it easier value to test against.